docs/index.rst

```diff
@@ -62,7 +63,7 @@ table, the only output topic in this example will be ``test-sqlite-jdbc-accounts
 Now, run the connector in a standalone Kafka Connect worker in another terminal (this assumes
 Avro settings and that Kafka and the Schema Registry are running locally on the default ports)::
 
-   $ ./bin/connect-standalone ./config/connect-standalone.properties quickstart-sqlite.properties
+   $ ./bin/connect-standalone ./config/connect-standalone.properties etc/kafka-connect-jdbc/quickstart-sqlite.properties
```

Comment:
while you are here... ./config/connect-standalone.properties does not exist.
We use etc/schema-registry/connect-avro-standalone.properties